.circleci/config.yml

```diff
@@ -200,7 +200,6 @@ workflows:
           requires:
             - helm_lint
             - unit_test
-            - integration_test
```

Comment:
We have an unresolved environment issue with integration tests inconsistently failing which we don't have time resources to look into currently. Removing this as a blocking step to deploy to dev for now due to approaching work deadlines.